src/main/scala/org/apache/mesos/chronos/scheduler/mesos/MesosTaskBuilder.scala

```diff
@@ -127,6 +128,13 @@ class MesosTaskBuilder @Inject()(val conf: SchedulerConfiguration) {
       }
     }
 
+    // Add maxCompletionTime
+    if (job.maxCompletionTime > 0L) {
+      val maxCompletionTime = DurationInfo.newBuilder().setNanoseconds(job.maxCompletionTime * pow(10, 9).toLong).build()
```

Comment:
I should have noticed this on the previous PR, but better late than never =).

I think we should use `scala.concurrent.duration.Duration` here, like this:

```
val maxCompletionTime = DurationInfo.newBuilder().setNanoseconds(Duration(job.maxCompletionTime.seconds.toNanos).build()
```